README.md

```diff
@@ -1,2 +1,105 @@
-# Receipe-app-rails
-The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Receipe-App
+
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app can allow you to make them public so anyone can access them.
+
+
+## Built With
+
+- Ruby on Rails [click here >>](https://guides.rubyonrails.org/)
+- Rubocop: [Rubocop](https://rubocop.org/)
+- UML schema diagram
+- Bootstrap
+- Ruby
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  Ruby
+  Ruby on Rails
+  PostgreSQL
+```
+### Setup
+Clone the repository and change the directory to blog
+
+``` 
+  $ git clone git@github.com:Favourezeugwa/Receipe-app-rails.git
+
+  $ cd Receipe-app-rails
+```
+
+### Install
+Install all project dependencies by running the command below
+ 
+``` 
+  $ bundle install
+```
+
+### Run the application
+```
+  Linux
+  $ bin/rails server
+```
+
+```
+  Windows
+  $ ruby bin/rails server
+```
+
+### Visit the homepage of application
+```
+  $ http://localhost:3000/
+```
+
+### Check linter errors
+``` 
+  $ rubocop .
+```
+
+### Run tests
+``` 
+  $ Rspec spec
+```
+
+### Authors
+
+👤 **Favour Amarachi Ezeugwa**
+
+- GitHub: [@Favourezeugwa](https://github.com/Favourezeugwa)
+- LinkedIn: [Favour Amarachi Ezeugwa](https://www.linkedin.com/in/favour-amarachi-ezeugwa-a5bb31149/)
+- Twitter:[@Favour_ezeugwa](https://twitter.com/Favour_ezeugwa)
+
+👤 **Ritobroto Mukherjee**
+
+- Portfolio: [@Rito](https://ritobroto-mukherjee.blogspot.com/)
+- GitHub: [@RitobrotoMukherjee](https://github.com/RitobrotoMukherjee)
+- Twitter: [@RitobrotoM3](https://twitter.com/RitobrotoM3)
+- LinkedIn: [Ritobroto Mukherjee](https://www.linkedin.com/in/ritobroto-mukherjee-519148ba/)
+
+👤 **Rich Adu-kyere**
+
+- GitHub: [@assadounto](https://github.com/assadounto)
+- Twitter: [@adukyerer](https://twitter.com/adukyerer)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/adu-kyere)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Favourezeugwa/Receipe-app-rails/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to [Microverse](https://bit.ly/MicroverseTN) for giving me this project to work on
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
### Optional
- Please fix this link for a professional readme file

Reply:
Okay will update it